sklearn/ensemble/_gb.py

```diff
@@ -335,6 +335,8 @@ def _init_state(self):
         # do oob?
         if self.subsample < 1.0:
             self.oob_improvement_ = np.zeros((self.n_estimators), dtype=np.float64)
+            self.oob_scores_ = np.zeros((self.n_estimators), dtype=np.float64)
+            self.oob_score_ = np.zeros((1), dtype=np.float64)
```

Comment:
Those need to be documented in the `Attributes` section of the public APIs' docstrings.